app/views/customers/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<section class="customer-info">
+  <h1><%= @customer.name %></h1>
+</section>
+
+<section class="item-info">
+  <% @customer.items.each do |item| %>
+    <p><%= "Item Name: #{item.name}" %></p>
+    <p><%= "Item Price: #{item.price}" %></p>
+    <p><%= "Supermarket: #{item.supermarket.name}" %></p>
+  <% end %>
+</section>
+
+<section class="add-item">
+    <%= form_with url: "/customer_items", method: :post, data: {turbo:false} do |form| %>
+      <%= form.hidden_field :customer_id, value: @customer.id %>
```

Comment:
Nice use of a hidden field to get this data to your controller. You could also add in additional params as an option in the form_with.